Content.Server.Database/ModelPostgres.cs

```diff
@@ -80,5 +80,14 @@ public override IQueryable<AdminLog> SearchLogs(IQueryable<AdminLog> query, stri
         {
             return query.Where(log => EF.Functions.ToTsVector("english", log.Message).Matches(searchText));
         }
+
+        public override int CountAdminLogs()
+        {
+            var command = new NpgsqlCommand("SELECT reltuples FROM pg_class WHERE relname = 'admin_log';", (NpgsqlConnection?) Database.GetDbConnection());
```

Comment:
You should put both the database command and the database connection in a using block here.